src/inferencer/inferencer.ts

```diff
@@ -7,14 +7,38 @@ export function inferProgram(program: es.Program): TypeAnnotatedNode<es.Program>
   function inferLiteral(literal: TypeAnnotatedNode<es.Literal>) {
     const valueOfLiteral = literal.value
     if (typeof valueOfLiteral === 'number') {
-      if (Number.isInteger(valueOfLiteral)) {
-        // annotate as int
-        literal.inferredType = {
-          kind: 'primitive',
-          name: 'integer'
-        }
-        literal.typability = 'Typed'
+      // if (Number.isInteger(valueOfLiteral)) {
+      // declare as int
+      literal.inferredType = {
+        kind: 'primitive',
+        name: 'integer'
```

Comment:
I think this should be `name: 'number'`, since we're doing away with integers

Reply:
Done